tasks/stacks/write-mixins-labels/task.yml

```diff
@@ -8,7 +8,9 @@ image_resource:
 inputs:
 - name: core-deps-ci
 - name: build-image-package-list
+  optional: true
 - name: run-image-package-list
+  optional: true
```

Comment:
Are these `optional`s left in for `fly execute` debugging?

Reply:
These are intentional, as they're not passed in for the Windows stack pipeline. If you look at the `run`, you'll see corresponding changes.